note/template/rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "nightly-2025-03-20"
+components = ["rustfmt", "rust-src"]
+targets = ["wasm32-wasi"]
```

Comment:
This should probably be changed to `wasm32-wasip1`, since AFAIK, the `wasm32-wasi` target is going to be removed (or maybe already has been? I can't recall)